manual/cops_rails.md

```diff
@@ -455,6 +455,30 @@ Name | Default value | Configurable values
 --- | --- | ---
 Include | `db/migrate/*.rb` | Array
 
+## Rails/CredentialsDirectAccess
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | Yes  | 2.0.1 | -
+
+This cop checks for access of app secrets and credentials.
+Direct access returns nil if the entry does not exist. This can happen
+with typo of entry name. Fetch raises an error upfront to avoid subtle
```

Comment:
The grammar here is a little off, how about "if there is a typo in the key name"?